README.md

```diff
@@ -1,40 +0,0 @@
-# WarpX Data
-
-## Overview
-
-This repository contains data files that are too large for the main [WarpX source directory](https://github.com/ECP-WarpX/WarpX).
-
-## Documentation
-
-This section describes the directory structure:
-
-...
-
-## Contributing
-
-Notes to follow.
-At the moment, just open PRs :)
-
-## Notes
-
-This repository might be rebased on a regular basis to minimize size by reduzing history bloat.
-
-## License
-
-WarpX Copyright (c) 2018-2021, The Regents of the University of California,
-through Lawrence Berkeley National Laboratory (subject to receipt of any
-required approvals from the U.S. Dept. of Energy).  All rights reserved.
-
-If you have questions about your rights to use or distribute this software,
-please contact Berkeley Lab's Innovation & Partnerships Office at
-IPO@lbl.gov.
-
-NOTICE.  This Software was developed under funding from the U.S. Department
-of Energy and the U.S. Government consequently retains certain rights. As
-such, the U.S. Government has been granted for itself and others acting on
-its behalf a paid-up, nonexclusive, irrevocable, worldwide license in the
-Software to reproduce, distribute copies to the public, prepare derivative
-works, and perform publicly and display publicly, and to permit other to do
-so.
```

Comment:
@ax3l how much of this should we keep with the license switch?

Reply:
Nothing. The old text was the software license, which does not apply for this repo.